README.md

```diff
@@ -34,7 +27,7 @@ These labs show you how to use ArcGIS Online to manage all of your data in the c
 * [Filter Data](/data/filter_data.md)
 * [Create Authenticated Traffic Layer](./data/create_authenticated_traffic_layer.md)
 
-### 2. Design
+### Design
 
 These labs illustrate how to use the Map viewer to create and style web maps. You can create new maps, select different basemaps, search for layers, use smart mapping to change symbol styles, style popups, label data, show/hide fields, filter data, save layers, create new layers and more. Creating a web map with great cartography is important as it serves as the foundation for building better mapping applications.
```

Comment:
since i haven't made any alterations here, that'd be something you might consider addressing in a seperate PR :smile: